.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "trailingComma": "es5"
+}
```

Comment:
I plan to do the same as in https://github.com/COVID2019-app/front-end/pull/39 for this project!